app/controllers/twitter_auth0_sign_in_callback.js

```diff
@@ -0,0 +1,78 @@
+const config = require('config')
+const request = require('request')
+const logger = require('../../lib/logger.js')()
+const { Authentication } = require('../../lib/auth0')
+
+var AccessTokenHandler = function (req, res) {
+  return function (err, response, body) {
+    if (err) {
+      console.error('Error obtaining access token from Twitter:')
+      console.log(err)
+      res.redirect('/error/no-twitter-access-token')
+      return
+    }
+
+    if (body.error && body.error === 'access_denied') {
+      res.redirect('/error/no-twitter-access-token')
+      return
+    }
+
+    let auth0 = Authentication()
```

Comment:
This should be a `const`. Anywhere a variable declared with `let` is never reassigned, use `const`.